editor-extensions/vscode/package.json

```diff
@@ -131,15 +131,18 @@
     "@typescript-eslint/parser": "^2.3.0",
     "eslint": "^6.4.0",
     "mocha": "^8.1.1",
-    "ovsx": "0.1.0-next.ae88c6d",
+    "ovsx": "^0.2.1",
     "prettier": "^2.2.1",
     "ts-loader": "^8.0.5",
     "typescript": "^4.0.2",
-    "vsce": "^1.83.0"
+    "vsce": "^1.100.1"
   },
   "__metadata": {
     "id": "b95eb54f-dbaf-4bb5-a18c-8b56da8a0829",
     "publisherDisplayName": "The Grain Programming Language",
     "publisherId": "28fcb3bd-39d4-48a2-9e5b-115ceb2c067b"
+  },
+  "dependencies": {
+    "chokidar": "^3.5.2"
```

Comment:
@marcusroberts why are you adding this dependency? I thought I saw that vscode has file watchers built into the client library.

Reply:
This is what I'm thinking of https://code.visualstudio.com/api/references/vscode-api#FileSystemWatcher

Reply:
Yes, but I think that's only available in the VSCode extension, not the LSP server.  Just checking.